apps/wallet-mobile/src/features/AddWallet/useCases/CreateWallet/AboutRecoveryPhrase.tsx

```diff
@@ -0,0 +1,91 @@
+import {useNavigation} from '@react-navigation/native'
+import {useTheme} from '@yoroi/theme'
+import * as React from 'react'
+import {StyleSheet, Text, View} from 'react-native'
+import {SafeAreaView} from 'react-native-safe-area-context'
+
+import {Button, Spacer, StatusBar} from '../../../../components'
+import {Space} from '../../../../components/Space/Space'
+import {CardAboutPhrase} from '../../common/CardAboutPhrase/CardAboutPhrase'
+import {LearnMoreButton} from '../../common/LearnMoreButton/LearnMoreButton'
+import {StepperProgress} from '../../common/StepperProgress/StepperProgress'
+import {useStrings} from '../../common/useStrings'
+
+export const AboutRecoveryPhrase = () => {
+  const {styles} = useStyles()
+  const strings = useStrings()
+  const navigation = useNavigation()
+  return (
+    <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.container}>
```

Comment:
if it has a SaveAreaView, it means it is getting the full screen, if it is getting the full screen it is a Screen -> AboutRecoveryPhraseScreen. otherwise drop the safe area if it is not.